### PR TITLE
Button

### PR DESCRIPTION
**Pull request by speirsluke**

### CODE REVIEW

File: src/Button/button.less [hunk 1]
```diff
@@ -0,0 +1,75 @@
+@import "../../styles/colors.less";
+
+button.button {
```
Comment (tomjbast) [inline src/Button/button.less]: why is this button.button? the button classname is just button right, why the double button?

File: src/Button/button.less [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    }
+    &:hover {
+      background-color: #f1edff;
+      border-color: #f1edff;
+    }
+  }
+  &:focus {
+    outline: none;
+  }
+}
```
Comment (tomjbast) [inline src/Button/button.less]: Where di you get all this CSS from? Did you have to copy a load of the CSS from dev tools, then copy it inot this files and remove bits you didnt want?

File: src/Button/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import MaterialButton from "@material-ui/core/ButtonBase"
+import cx from "classnames"
+
+import "./button.less"
+
+
+const Button = props => {
+    const classes = cx("button", props.varient, {
+      inactive: props.inactive
+    })
```
Comment (tomjbast) [inline src/Button/index.js]: This is a lack of my knowledge of cx lib, but whats happening here? 

the default classname is button? Whats props.variant (also you spelt variant wrong I think)

File: src/Button/button.less (hunk 1 above)
Comment (speirsluke) [reply src/Button/button.less]: To increase specificity so it can override some of the styling set by MUI

File: src/Button/button.less (hunk 2 above)
Comment (speirsluke) [reply src/Button/button.less]: For one or two bits, for the most part its just overriding the default CSS that the button has

File: src/Button/index.js (hunk 3 above)
Comment (speirsluke) [reply src/Button/index.js]: So button and props.**_variant_** are two default classes, and then if the button has the inactive prop it also gets the inactive class

**Pull request closed without merging**